nixos/modules/services/web-apps/immich.nix

```diff
@@ -232,6 +232,10 @@ in
 
     services.postgresql = mkIf cfg.database.enable {
       enable = true;
+      package =
+        lib.warnIf (lib.versionOlder "1.135.3" pkgs.immich.version)
+          "nixos/immich: the services.postgresql.packge override may not be needed anymore."
```

Comment:
This feels a bit weird because it prints out non-actionable warnings to users after the next upgrade.

No strong opinion though, will defer to the maintainers.

Reply:
Wouldn't that break if somebody decides to test out the package after release via an overlay for instance?

But again, I'm defering to the maintainer because I'm not sure what to do (my stance is that it's the maintainer's job to make sure the module is kept up-to-date - for better or worse).

Reply:
Let's not add this. I'll remember to check.